scipy/stats/_hypotests.py

```diff
@@ -1803,22 +1803,18 @@ def confidence_interval(self, confidence_level=.95):
 
         if not 0 < confidence_level < 1:
             raise ValueError("Confidence level must be between 0 and 1.")
-        # determine the critical value of the studentized range using the
-        # appropriate confidence level, number of treatments, and degrees
-        # of freedom as determined by the number of data less the number of
-        # treatments. ("Confidence limits for Tukey's method")[1]. Note that
-        # in the cases of unequal sample sizes there will be a criterion for
-        # each group comparison.
-        params = (confidence_level, self._nobs, self._ntreatments - self._nobs)
+        # ref.2 p.6
+        # H0 was rejected if $|v| \ge q(\alpha, K, \nu)/\sqrt{2}$
```

Comment:
Please preserve the essential information in both notes. Continue to refer to the reference number within square brackets as in the original comment, like `[1]`. Something like:

> determine the critical value of the studentized range using the appropriate confidence level, number of treatments, and degrees of freedom. See [1] "Confidence limits for Tukey's method" / [2] p. 6 "H0 was rejected" for Games Howell

For "H0 was rejected", just include whatever information helps the reader find the relevant section of the paper.